tailwind.config.js

```diff
@@ -0,0 +1,19 @@
+const plugin = require("tailwindcss/plugin");
+
+module.exports = {
+  purge: [],
```

Comment:
We should be able to add `mode: "jit"` here. And we'll need to configure the purge option, something like:
```
purge: {
    enabled: true,
    content: [
      "./app/views/**/*.html.haml",
      "./app/assets/javascript/templates**/*.js"
    ]
  }
```